.github/workflows/cd.yml

```
@@ -0,0 +1,86 @@
+          host: ${{ secrets.SSH_HOST }} # EC2 퍼블릭 IPv4
+          username: ${{ secrets.SSH_USERNAME }} # ubuntu
+          key: ${{ secrets.SSH_KEY }}
+          port: 22
+          # docker-compose가 있는 디렉토리로 이동
+          script: |
+            cd /home/ubuntu/troublog
+            sudo docker login -u ${{ secrets.DOCKER_USERNAME }} -p ${{ secrets.DOCKER_PASSWORD }}
+            sudo docker-compose pull       # 새 이미지 받기
+            sudo docker-compose down || true  # 컨테이너 종료
+            sudo docker-compose up -d # 재실행
+            sudo docker image prune -f # 안 쓰는 이미지 정리
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)